File: DSA/SuryanshShah_590013441/Day5Question1.java
import java.util.*
public class Day5Question1{
    public static void main(String[] args){
    System.out.println("Enter the no. of elements in the aaray:")
    Scanner scr=new Scanner(System.in);
    int n=scr.nextInt();
    int [] array=new int[n];
    int i;
    System.out.println("Enter the elements into the array:");
    for(i=0;i<n;i++)
    {
        array[i]=scr.nextInt();
    }
    int j,counter=0;
    for(i=0;i<n;i++)
    {
        counter=0;
        for(j=0;j<n;j++)
        {
            if(array[i]==array[j])
                counter++;
        }
        if(counter==1)
            System.out.println(array[i]+" is a Unique element");
    }
}

File: DSA/Shreya_Goyal_590015491/Day_5_Question_1.c
#include <stdio.h>

int main() {
    int arr[5],i,j;

printf("Enter 5 numbers:\n");
for(i=0; i<5; i++) {
    scanf("%d", &arr[i]);
}

for(i=0; i<5; i++) {
    int c=0;

    for(j=0; j<5; j++) {
        if(arr[i] == arr[j]) {
            c++;
        }
    }

    if(c==1) {
        printf("Unique element: %d\n", arr[i]);
        break;
    }
}

return 0;
}
